src/public/app/widgets/type_widgets/geo_map.ts

```diff
@@ -260,8 +260,8 @@ export default class GeoMapTypeWidget extends TypeWidget {
     #buildIcon(bxIconClass: string, colorClass: string, title: string) {
         return this.L.divIcon({
             html: `\
-                <img class="icon" src="${asset_path}/node_modules/leaflet/dist/images/marker-icon.png" />
-                <img class="icon-shadow" src="${asset_path}/node_modules/leaflet/dist/images/marker-shadow.png" />
+                <img class="icon" src="${asset_path}/app-dist/leaflet/images/marker-icon.png" />
+                <img class="icon-shadow" src="${asset_path}/app-dist/leaflet/images/marker-shadow.png" />
```

Comment:
I think hard-coding `app-dist` is not a good approach since it seems to fail when run using `npm run server:start`, most likely because the webpack build was not run.

I still get an icon not found. I'm going to try a fix for it that involves not having to manually deal with the image and let webpack do its magic.